docs/marionette.functions.md

```diff
@@ -124,16 +126,35 @@ var Obj = Mn.Object.extend({
 
   initialize: function(attributes, options){
```

Comment:
actually this was incorrect to begin with.  `attributes` isn't a thing.  That'd be true for a `Backbone.Model` but not for an object...